### DISCUSSION TITLE
How can I evaluate my model?

### DISCUSSION THREAD

**Discussion by elcronos** [PyTorchLightning/lightning-flash#234] [Q&A] [answered]
#### What is your question?
I followed the tutorial for custom task classification and passed 3 folders corresponding to train, test and validation. When I train the model I can see printed at the end the train accuracy and validation accuracy. However, I cannot see what's the test accuracy. Also, when I load the model I'd like to test with another dataset. What's the best way of evaluation ?

**Comment by satishjasthi** [comment, marked as answer]
Hi elcronos 

You can evaluate your model on a test data set using test method on the trainer as shown [here](https://lightning-flash.readthedocs.io/en/latest/reference/image_classification.html#finetuning)
Furthermore, if you want to test your model on other datasets then use the inference mode of the model as show [here](https://lightning-flash.readthedocs.io/en/latest/reference/image_classification.html#inference)

Hope this clarifies your doubt

**Comment by elcronos** [comment]
Thanks
